WidgetKit/SearchQuickLinksSmall/SmallQuickLink.swift

```diff
@@ -46,12 +44,12 @@ struct SmallQuickLinkWidget: Widget {
     private let kind: String = "Quick Actions - Small"
 
     public var body: some WidgetConfiguration {
-        IntentConfiguration(kind: kind, intent: QuickActionIntent.self, provider: IntentProvider()) { entry in
+        StaticConfiguration(kind: kind, provider: IntentProvider()) { entry in
             SmallQuickLinkView(entry: entry)
         }
         .supportedFamilies([.systemSmall])
-        .configurationDisplayName(String.QuickActionsGalleryTitle)
-        .description(String.QuickActionGalleryDescription)
+        .configurationDisplayName(String.QuickActionsGalleryTitlev2)
```

Comment:
Can we keep the new title `QuickActionsGalleryTitle`

Reply:
Actually wait, let's use whichever one is the newest... I forgot which one it was